app/controllers/api/v1/public/publishers_controller.rb

```diff
@@ -1,5 +0,0 @@
-class Api::V1::Public::PublishersController < Api::V1::Public::BaseController
```

Comment:
Can you remove route that went to this controller?

Reply:
It's already removed.